themes/root/templates/HelpTranslations/de/blender_advsearch.phtml

```diff
@@ -5,7 +5,7 @@
   <dd>
     <p>Wenn Sie das erste Mal die erweiterte Suche aufrufen, erscheinen
        verschiedene Suchfelder. In jedem Feld können Sie einen Suchbegriff
-       eingeben. Folgende <a href="<?=$this->url('help-home', [], ['query' => ['topic' => 'blender']])?>">Suchoperatoren</a>
+       eingeben. Folgende <a href="<?=$this->url('help', [], ['query' => ['topic' => 'blender']])?>">Suchoperatoren</a>
```

Comment:
This still seems to be using the query approach. Might be worth searching the templates for `['query' => ['topic'` to see if similar things were missed anywhere else.